openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/KpiRepository.java

```diff
@@ -225,14 +225,15 @@ public KpiUpdater(Kpi original, Kpi updated, Operation operation) {
 
     @Override
     public void entitySpecificUpdate() throws IOException {
-      updateFromRelationships(
+      updateToRelationships(
```

Comment:
Why have we done this change from `updateFromRelationships` to `updateToRelationships`? The PR says: `add KPI resource test`.

Reply:
Reviewed with @mohityadav766